curriculum/challenges/english/25-front-end-development/quiz-css-layout-and-effects/66ed8ff4f45ce3ece4053eb4.md

```diff
@@ -325,131 +325,131 @@ Which CSS property is used to apply transformations such as rotation, scaling, a
 
 #### --text--
 
-How is the total width of an element calculated in CSS?
+How is the total width of an element calculated in CSS when using the `border-box` value for the `box-sizing` property?
 
 #### --distractors--
 
-Total width = width + margin + padding + border
+Total Width = Content Width + Border + Margin
 
 ---
 
-Total width = width + margin + padding
+Total Width = Content Width + Padding + Border + Margin
 
 ---
 
-Total width = width + margin + border
+Total Width = Content Width
 
 #### --answer--
 
-Total width = width + left padding + right padding + left border + right border
+Total Width = Content Width + Padding + Border
 
 ### --question--
 
 #### --text--
 
-What does the filter property in CSS do?
+What is the purpose of the `filter` property in CSS?
 
 #### --distractors--
 
-It changes the layout of the element on the page
+Changing the scale and rotation of an element
 
 ---
 
-It changes the background color and texture of an element
+Changing the background color and texture of an element
 
 ---
 
-It alters the size and position of an element
+Altering the size and position of an element
 
 #### --answer--
 
-It applies visual effects like blur and saturation to an element
+Applying graphical effects, like blur and contrast, to an element
```

Comment:
By mentioning blur and contrast in the correct answer, it makes it too easy to identify since these are well-known filter effects. We could generalize the answer. For example:
`Modifying the visual appearance of an element using various graphical effects.`